packages/main/src/plugin/learning-center/guides.json

```diff
@@ -0,0 +1,44 @@
+    },
+    {
+      "id": "spring",
+      "url": "https://developers.redhat.com/articles/2023/07/03/managing-java-containers-quarkus-and-podman-desktop",
```

Comment:
shouldn't it be https://developers.redhat.com/articles/2023/10/19/containerize-spring-boot-application-podman-desktop